site/docs/reference/Connectors/capture-connectors/netsuite.md

```diff
@@ -18,7 +18,65 @@ Flow captures collections from any NetSuite object to which you grant access dur
 
 ## Setup
 
-Create a custom role estuary-role following [these steps](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/bridgehead_N286284.html). Any objects to which you grant the estuary-role access will be captured.
+Step 1: Create NetSuite account
+ - Create account on [Oracle NetSuite](https://www.netsuite.com/portal/home.shtml)
+ - Confirm your Email
+Step 2: Setup NetSuite account
```

Comment:
Suggestion for readability -- it might be easier on the eyes to add an extra space between the steps, or else text tends to render kind of smushed together.